src/renderer/store/help.js

```diff
@@ -17,6 +17,7 @@ export const defaultFileState = {
     isBom: false
   },
   lineEnding: 'lf', // lf or crlf
+  trimTrailingNewline: 3,
```

Comment:
Why the default value is 3, not 2 as in other place?

Reply:
Because we can only detect the final newline type from an existing file but not for new files, so we keep the current behavior and do nothing.